test/local_detuning.jl

```diff
@@ -0,0 +1,61 @@
+using Braket, Test, JSON3, StructTypes, Mocking, UUIDs, DecFP
+
+Mocking.activate()
+
+struct MockRydbergLocal
```

Comment:
Are these mocks needed? Why can't the already implemented structs be used?

Reply:
Removed `MockDiscretizationProperties.` as we can used `Braket.DiscretizationProperties.` Also, used     `Braket.RydbergGlobal` instead.

Since `Braket.RydbergLocal` is not defined, we cannot use it directly which is a primary reason for defining  mock struct `(MockRydbergLocal) `which provides ` (timeResolution, commonDetuningResolution, localDetuningResolution)`